### PR TITLE
feat: make deployer only answer its own project

### PR DESCRIPTION
**Pull request by chesedo**

### CODE REVIEW

**Review by oddgrd — APPROVED** [review]
Looks good! Just one question from a tracing noob: does this add the project name to the tracing layer?

**Comment by chesedo** [comment]
@oddgrd, not yet. I'll spin up another PR for a bunch of tracing stuff

File: gateway/src/project.rs [hunk 1]
```diff
@@ -615,9 +615,7 @@ pub struct Service {
 
 impl Service {
     pub fn from_container(container: ContainerInspectResponse) -> Result<Self, ProjectError> {
-        let container_name = safe_unwrap!(container.name.strip_prefix("/")).to_string();
-
-        let resource_name = safe_unwrap!(container_name.strip_suffix("_run")).to_string();
+        let name = safe_unwrap!(container.config.labels.get("project.name")).to_string();
```
Comment (chesedo) [inline gateway/src/project.rs:618]: @brokad I tried to trace where `Service::name` will be used but it seems to be only for the health check. However, can you still confirm I'm not breaking anything here?

Comment (brokad) [reply gateway/src/project.rs:620]: You may be breaking existing containers that were not deployed with the label.

Comment (chesedo) [reply gateway/src/project.rs]: Thanks @brokad!! I've changed it to a backwards compatible implementation

**Pull request merged**